### ISSUE TITLE
Feature to vertically reorder timetables

### ISSUE THREAD

**Issue by chancecorbeil** [Trillium-Solutions/transit-custom-posts#48]
Timetables without legends are stacked vertically. Build a tool on the route post editor to change the timetable stack order.

**Comment by chancecorbeil** [comment]
This feature is set up and ready to use. 

Originally, timetables are sorted by the 'timetable id'. This is fine for when the timetable order doesn't matter. This is still the default functionality. 

To have more control over vertical timetable order, there is now a new field on the Route editor to designate how to order the timetables. This field is 'Order by' and the options are 'Timetable ID Field' and 'Timetable Order Field'.

If 'Timetable Order Field' is selected, then then timetables will be ordered by a custom field 'Timetable Order' on the Timetables editor. The 'Timetable Order' field uses numbers to order the timetables.

To summarize:
To reorder timetables vertically, set the route to 'Order by': 'Timetable Order Field' and set the timetables 'Timetable Order' field in the numerical order you want the timetables to appear.
